src/pages/map/map.ts

```diff
@@ -22,12 +24,12 @@ export class MapPage {
   }
 
   initNativeMaps(mapEle) {
-    this.map = new GoogleMap(mapEle);
+    this.googleMap = new GoogleMap(mapEle);
     mapEle.classList.add('show-map');
 
-    GoogleMap.isAvailable().then(() => {
-      const position = new GoogleMapsLatLng(43.074395, -89.381056);
-      this.map.setPosition(position);
+    this.googleMaps.isAvailable().then(() => {
```

Comment:
Likely. I will make the change. However, I did not see maps in the app. There may still be config issues or operator error.